src/Tizen.NUI.Components/Attributes/ViewAttributes.cs

```diff
@@ -25,14 +27,25 @@ namespace Tizen.NUI.Components
     [EditorBrowsable(EditorBrowsableState.Never)]
     public class ViewAttributes : Attributes
     {
+        static public T CreateView<T>(string style) where T : View, new()
+        {
+            T ret = new T();
+            (Tizen.NUI.Components.StyleManager.Instance.GetAttributes("style") as ViewAttributes).ApplyToView(ret);
+            return ret;
+        }
+
         /// <summary>
         /// Construct ViewAttributes.
         /// </summary>
         /// This will be public opened in tizen_5.5 after ACR done. Before ACR, need to be hidden as inhouse API.
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public ViewAttributes() : base() { }
+        public ViewAttributes() : base()
+        {
+            StyleManager.Instance.ThemeChangedEvent += OnThemeChangedEvent;
```

Comment:
IMHO, if you don't subscribe OnThemeChangedEvent from the StyleManager instance, the ViewAttribute instance will never be able to be collected by GC.

Reply:
Will remove it in Dispose().